tests/test_integration/test_code_generation.py

```diff
@@ -24,21 +24,13 @@
 from gt4py import backend as gt_backend
 from gt4py import storage as gt_storage
 
-
 from .stencil_definitions import REGISTRY as stencil_definitions
 from .stencil_definitions import EXTERNALS_REGISTRY as externals_registry
+from ..definitions import ALL_BACKENDS, CPU_BACKENDS, GPU_BACKENDS, INTERNAL_BACKENDS
 
 
 @pytest.mark.parametrize(
-    ["name", "backend"],
-    itertools.product(
-        stencil_definitions.names,
-        [
-            name
-            for name in gt_backend.REGISTRY.names
-            if gt_backend.from_name(name).storage_info["device"] == "cpu"
-        ],
-    ),
+    ["name", "backend"], itertools.product(stencil_definitions.names, CPU_BACKENDS)
 )
```

Comment:
According to https://github.com/pytest-dev/pytest/issues/815, we could do 

```suggest
@pytest.mark.parametrize("name", stencil_definitions.names)
@pytest.mark.parametrize("backend", CPU_BACKENDS)
def test_generation_cpu(name, backend):
```

Reply:
I think it's a good idea. Let's create an issue for this (#43) and try change it everywhere in a different PR